### ISSUE TITLE
Add OpenROAD icon to Ubuntu task bar

### ISSUE THREAD

**Issue by oharboe** [The-OpenROAD-Project/OpenROAD#6173]
### Description

When OpenROAD is minimized, I get a generic icon, would be nice with an OpenROAD icon to easily spot OpenROAD:

![image](https://github.com/user-attachments/assets/0098e633-6f90-47f8-8e3d-7ab0278db3ac)

### Suggested Solution

Add icon

**Comment by gadfort** [comment]
@maliberty I can take a look at this since I've noticed this as well.

**Comment by maliberty** [comment]
I'm using 22.04 LTS fwiw

**Comment by gadfort** [comment]
@maliberty I'm on 24.04, which seems to rely on a slightly different system.

**Comment by oharboe** [comment]
Still no icon:

![image](https://github.com/user-attachments/assets/50155f62-8345-4b39-8f28-abd17ef669bd)

**Comment by gadfort** [comment]
@oharboe if OpenROAD is installed in a non-standard location, the desktop file will not be in the correct place and will need to copied to the correct place. I suspect this is an issue with your installation

**Comment by oharboe** [comment]
> @oharboe if OpenROAD is installed in a non-standard location, the desktop file will not be in the correct place and will need to copied to the correct place. I suspect this is an issue with your installation. If you are installing using orfs, then it's an orfs issue

My usecase is to build and run OpenROAD without installing it into the OS.

Can icons work with relative paths?

**Comment by gadfort** [comment]
@oharboe no, the .desktop file must appear in a particular set of directories (as per freedesktop). This is a limitation of the Ubuntu desktop system. You will need the ORFS installer to copy that file to the correct place (`$home/.local/share/applications`) is sufficient. The location of the icon itself doesn't matter.

**Comment by oharboe** [comment]
I see. In that case, I dont see an icon as possible/practical for my usecase.
